model_analyzer/config/generate/run_config_generator_factory.py

```diff
@@ -70,20 +70,16 @@ def create_run_config_generator(
             new_models.append(
                 ModelProfileSpec(model, command_config, client, gpus))
 
-        ensemble_composing_models = RunConfigGeneratorFactory._create_ensemble_composing_models(
+        composing_models = RunConfigGeneratorFactory._create_composing_models(
             new_models, command_config, client, gpus)
 
-        bls_composing_models = RunConfigGeneratorFactory._create_bls_composing_models(
-            command_config, client, gpus)
-
         if (command_config.run_config_search_mode == "quick" or
-                bls_composing_models):
+                composing_models):
```

Comment:
is this needed? Is there not protection elsewhere for composing models with !quick?